packages/components/src/components/contract-card/result-overlay/result-overlay.tsx

```diff
@@ -88,7 +88,9 @@ const ResultOverlay = ({
                         <span
                             id={`dc_contract_card_${contract_id}_result_close_icon`}
                             className='dc-result__close-btn'
-                            onClick={() => onClickRemove?.(contract_id)}
+                            onClick={() => {
+                                if (contract_id) onClickRemove?.(contract_id);
```

Comment:
`() => contract_id && onClickRemove?.(contract_id);`
Would it work?

Reply:
Or this is a bad practise?

Reply:
what you proposed will return a boolean i think. `() => boolean` instead of `() => void` with my solution

Reply:
i think we should be careful with using `&&` as `if` in javascript logic and not to get confused with short circuiting a react component. `something &&  <SomeComponent />` is not the same as `something && function call` they will both return boolean but short circuiting a component is ok because we are dealing with rendering.

Reply:
Okay, thank you, I think you'll right and it'll be safer to go with `if`